redis/redis.go

```diff
@@ -105,6 +138,20 @@ func DoWithTimeout(c Conn, timeout time.Duration, cmd string, args ...interface{
 	return cwt.DoWithTimeout(timeout, cmd, args...)
 }
 
+// ReceiveContext receives a single reply from the Redis server.
+// This function will use the timeout in this rule:
```

Comment:
I think it would be better to talk about context generically here an example of this kind of thing can be seen here:
https://golang.org/pkg/net/http/#Server.Shutdown

In addition this should be describing the behaviour not the implementation .

Reply:
take a look at the current comment.